relay-general/src/protocol/thread.rs

```diff
@@ -125,6 +125,10 @@ pub struct Thread {
     /// A flag indicating whether the thread was responsible for rendering the user interface.
     pub main: Annotated<bool>,
 
+    /// Thread state at the time of the crash.
+    #[metastructure(skip_serialization = "empty")]
+    pub state: Annotated<String>,
```

Comment:
Now that I'm thinking about it, it's probably best to make this an `enum` instead. From your PR description, I assume you're focusing mostly on Android for now, and you link to two places where there's a list of possible values. We can start from these.